x-pack/solutions/chat/plugins/workchat-app/tsconfig.json

```diff
@@ -39,6 +39,7 @@
     "@kbn/wci-browser",
     "@kbn/features-plugin",
     "@kbn/wc-genai-utils",
+    "@kbn/workchat-app",
```

Comment:
once you resolved the absolute import, this isn't needed

Reply:
addressed in https://github.com/elastic/kibana/pull/217650/commits/20bae1edb5ef7ca9da2da5d904f1eeab1ffa4770